benchmarks/ner_embeddings/scripts/constants.py

```diff
@@ -0,0 +1,12 @@
+# A mapping of datasets and their vectors
+DATASET_VECTORS = {
+    "archaeo": {"spacy": "nl_core_news_lg", "fasttext": "fasttext-nl", "lang": "nl"},
+    "anem": {"spacy": "en_core_web_lg", "fasttext": "fasttext-en", "lang": "en"},
+    "es-conll": {"spacy": "es_core_news_lg", "fasttext": "fasttext-es", "lang": "es"},
+    "nl-conll": {"spacy": "nl_core_news_lg", "fasttext": "fasttext-nl", "lang": "nl"},
+    "wnut17": {"spacy": "en_core_web_lg", "fasttext": "fasttext-en", "lang": "en"},
+    "finer": {"spacy": "fi_core_news_lg", "fasttext": "fasttext-fi", "lang": "fi"},
```

Comment:
I think `finer` is English and not Finnish right?
```suggestion
    "finer": {"spacy": "en_core_web_lg", "fasttext": "fasttext-en", "lang": "en"},
```

Reply:
Hmm, I did check the dataset from here too: https://github.com/mpsilfve/finer-data/blob/master/data/digitoday.2015.test.csv and it seems that they're finnish? Either way, we didn't include finer in the results so we might as well remove it? What do you think?